sources/Inputs/include/Configuration.h

```diff
@@ -116,16 +116,31 @@ class Configuration {
     return dsoVoltageLevel_;
   }
 
+  /**
+   * @brief type of active power compensation for generator
+   */
+  enum class ActivePowerCompensation { P, TARGET_P, PMAX };
```

Comment:
Each possible value should be commented

Reply:
@marcochiaramello Any ideas on what the comments would look like for P, TARGET_P and PMAX ?

Reply:
Something like:
P -> active power mismatch compensation proportional to active power injection P
TARGET_P -> active power mismatch compensation proportional to targetP
PMAX -> active power mismatch compensation proportional to generator maximal active power PMax

@cossartque any opinion on this?
Thanks

Reply:
let's go for this then